app/javascript/presenters/TaskPresenter.js

```diff
@@ -11,3 +11,13 @@ export default new PropTypesPresenter(
     state: PropTypes.string,
   },
 );
+
+export const STATES = [
+  { key: 'new_task', value: 'New' },
+  { key: 'in_development', value: 'In Dev' },
+  { key: 'in_qa', value: 'In QA' },
+  { key: 'in_code_review', value: 'in CR' },
+  { key: 'ready_for_release', value: 'Ready for release' },
+  { key: 'released', value: 'Released' },
+  { key: 'archived', value: 'Archived' },
+];
```

Comment:
mb
```
export const STATES = { 'new_task' => 'New', ... }
```

Reply:
Mb no? I don't want to have a collision.